<launch>
    <node pkg="joy" type="joy_node" name="joy_node"/>
    <node pkg="controller" type="controller_node" name="controller_node" output="screen">
        <param name="accel" type="double" value="0.2"/>
        <param name="steer_offset" type="double" value="0.05"/>

        <param name="p_gain" type="double" value="0.7"/>
        <param name="i_gain" type="double" value="0.005"/>
        <param name="d_gain" type="double" value="2"/>

        <param name="menual_start" type="bool" value="false"/>
    </node>
</launch>
